numpy/core/src/umath/ufunc_object.c

```diff
@@ -5997,18 +5998,19 @@ ufunc_get_doc(PyUFuncObject *ufunc)
      * introspection on name and nin + nout to automate the first part
      * of it the doc string shouldn't need the calling convention
      */
-    doc = PyObject_CallFunctionObjArgs(
-        _sig_formatter, (PyObject *)ufunc, NULL);
+    doc = PyObject_CallFunctionObjArgs(_sig_formatter,
+                                       (PyObject *)ufunc, NULL);
     if (doc == NULL) {
         return NULL;
     }
+    // FIXME: check if ufunc docstrings can/should be UTF-8
```

Comment:
I don't understand the quetion behind this comment - the PR hasn't changed anything, has it?

Reply:
```suggestion
```

Reply:
Done.